src/test/java/duke/DukeTest.java

```diff
@@ -0,0 +1,12 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Perhaps the position of these 2 statements should be reversed (so that the ordering is consistent with that of other files)?